specs/core/0_beacon-chain.md

```diff
@@ -226,54 +231,58 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 
 ## Data structures
 
-### Deposits
+### Beacon chain operations
 
-#### `DepositParametersRecord`
+#### Proposer slashings
+
+##### `ProposerSlashingRecord`
 
 ```python
 {
-    # BLS pubkey
-    'pubkey': 'uint384',
-    # BLS proof of possession (a BLS signature)
-    'proof_of_possession': ['uint384'],
-    # Withdrawal credentials
-    'withdrawal_credentials': 'hash32',
-    # Initial RANDAO commitment
-    'randao_commitment': 'hash32',
+    # Proposer index
+    'proposer_index': 'uint24',
+    # First proposal data
+    'proposal_data_1': ProposalSignedData,
+    # First proposal signature
+    'proposal_signature_1': '[uint384]',
+    # Second proposal data
+    'proposal_data_2': ProposalSignedData,
+    # Second proposal signature
+    'proposal_signature_2': '[uint384]',
 }
 ```
 
-### Beacon chain blocks
+#### Casper slashings
 
-#### `BeaconBlock`
+##### `CasperSlashingRecord`
```

Comment:
I would agree with Vitalik that we should keep `Record` for objects stored in the state. What about `CasperSlashingObject` or `CasperSlashingSpecial`?

Reply:
In general object is implicit on all objects... but due to the conflicting debate on naming these things, I'll go with object for now.

Reply:
I actually just removed the suffix from the objects. I think it is reasonable.